src/main/java/com/lophita/tobaskincare/rest/StockController.java

```diff
@@ -43,4 +39,34 @@
                 }).collect(Collectors.toList());
         return stockDtoList;
     }
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    @ResponseBody
+    public StockDto addStock(@Valid @RequestBody StockDto stockDto) {
```

Comment:
kalau mw balikin response dr sebuah endpoint best practice nya itu
bentukan data nya mesti nya
`"code":"SUCCESS",
"message":"Success",
"data":{}
"errors":null -> list of string tipe datanya`

jadi bikin aja satu class namanya `BaseResponse` dgn attribute diatas, tp bagian yg data tipe datanya generic.
jd nnti balikan di controller nya BaseResponse<StockDto>